src/backend/downloadmanager/utils.ts

```diff
@@ -9,9 +9,9 @@ import { DMStatus, InstallParams, Runner } from 'common/types'
 import i18next from 'i18next'
 import { notify, showDialogBoxModalAuto } from '../dialog/dialog'
 import { isOnline } from '../online_monitor'
-import { fixesPath, isWindows } from 'backend/constants'
-import path from 'path'
-import { existsSync, mkdirSync } from 'graceful-fs'
+import { fixesPath, gogdlConfigPath, isWindows } from 'backend/constants'
+import pathModule from 'path'
```

Comment:
I had to rename this to `pathModule` cause `path` already exists inside the `installQueueElement` function defined as a variable out of the install params